style.css

```diff
@@ -1,3 +1,16 @@
 body {
   background-color: aqua;
-}
\ No newline at end of file
+}
+body {
+    margin: 0;
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
+      'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+      sans-serif;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+}
```

Comment:
I think you need to use the required CSS style for this file. this style are not required based on the assignment.

Reply:
- [ ] Good use of inline comments here. Your analysis looks ambiguous and may further confuse the student. It would help if you pointed out what is wrong and what the student should do to fix it. 🥲